### PR TITLE
feat(`Framework`): `commands_vector` function

### PR DESCRIPTION
**Pull request by Fyko**
This PR:
- creates `Framework#commands_vector`, a function to create a vector of `Commands`, than can be passed into `serde_json` to create a JSON string
- uses `[workspace.dependencies]` in the root for some reused dependencies

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -446,3 +446,44 @@ shown to the user using the `#[modal(..)}` attributes. To see the full list of a
...
+# Bulk Commands Overwrite
+If you'd like to use Discord's [Bulk Overwrite Global Application Commands](https://discord.com/developers/docs/interactions/application-commands#bulk-overwrite-global-application-commands) enpoint, perhaps in tandem with a [commands lockfile](https://github.com/carterhimmel/thoth/tree/28c3855b1c55c9ed839bbbcbf9e9c704bf2bd81a/.github/workflows/cd_commands.yml), you'll want to use `Framework#commands_vector`.
+
+> **Note**
+> This requires the `bulk` feature.
+
+```rust
+type MyFramework = Arc<Framework<()>>;
+
+async fn create_framework(
```
Comment (AlvaroMS25) [inline README.md]:
```suggestion
fn create_framework(
```
function does not need to be async here

File: zephyrus/src/framework.rs [hunk 2]
```diff
@@ -402,6 +402,49 @@ where
         Ok(commands)
     }
 
+    /// Creates a vector of Command objects, to be used against Discord's bulk endpoint.
+    #[cfg(feature = "bulk")]
+    pub fn commands_vector(
```
Comment (AlvaroMS25) [inline zephyrus/src/framework.rs]: maybe change the name to something like `twilight_commands`?

File: README.md [hunk 3]
```diff
@@ -446,3 +446,44 @@ shown to the user using the `#[modal(..)}` attributes. To see the full list of a
...
+Zephyrus [doesn't quite support message and user commands](https://github.com/AlvaroMS25/zephyrus/issues/6).  
+
+If you want to:
+- use message/user commands
+- utilize [Bulk Overwrite Global Application Commands]()
+- and utilize a [Commands Lockfile]()  
+
+you'll have to:
+- selectively filter which `INTERACTION_CREATE` [types](https://docs.rs/twilight-model/0.15.2/twilight_model/application/interaction/enum.InteractionType.html) are fed to Zephyrus
+- exclusively use Zephyrus' modal system or your own, not both -->
```
Comment (Fyko) [inline README.md:489]: @AlvaroMS25
 I was going to use this section to show how the new function could be used to support user and message commands in the meantime. eh?

Comment (AlvaroMS25) [reply README.md:489]: I think is a good idea while message/user commands get support, it'll help the user know proper alternatives

**Comment by AlvaroMS25** [comment]
Looks good! Merging to `master` branch!

**Pull request merged**